rdtools/plotting.py

```diff
@@ -340,7 +340,7 @@ def availability_summary_plots(power_system, power_subsystem, loss_total,
     ax3.set_ylabel('Estimated lost power [kW]')
 
     # cumulative energy
-    ax4.plot(energy_cumulative, label='Reported Production')
+    energy_cumulative.plot(ax=ax4, label='Reported Production')
```

Comment:
For future readers:  this change has something to do with newer versions of pandas (e.g. 1.1.0) converting datetimes in an incompatible way with matplotlib, so mixing `data.plot(ax=ax)` with `ax.plot(data)` resulted in bad behavior.